SampleBrowser/SampleBrowser/Model/RootViewControllerModel.swift

```diff
@@ -97,10 +103,12 @@ class RootViewControllerModel {
             urlTextFieldA11yId: "urlTextField",
             urlTextFieldA11yLabel: "Address Bar",
             searchEngineImage: UIImage(named: "bingSearchEngine"),
-            lockIconImageName: StandardImageIdentifiers.Medium.lock,
+            lockIconImageName: StandardImageIdentifiers.Medium.sun,
```

Comment:
Why did the lock icon change here? I think displaying a sun to show if a website is secure or not is kinda weird 😆

Reply:
Now the lock is in the Large struct since is missing in Medium